charter.html

```diff
@@ -75,17 +75,28 @@
       <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1>
       <!-- delete PROPOSED after AC review completed -->
 
-      <p class="mission">The <strong>mission</strong> of the <a href=""Private Advertising Technology Working Group</a>
-         motivated by the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>, is to
-         specify web features and APIs that support advertising while
-         acting in the interests of users, in particular providing strong
-         privacy assurances.  The Working Group welcomes participation from
-         browser vendors, OS vendors, mobile application vendors, advertisers,
-         publishers, ad buyers, advertising platforms and intermediaries,
-         privacy advocates, web application developers, and other interested
-         parties.
+      <p class="mission">The <strong>mission</strong> of the 
+        <a href="#">Private Advertising Technology Working Group</a>
+        motivated by the need to develop open standards supporting the development of the Open Web in accordance with 
+        the <a href="https://www.w3.org/2009/12/Member-Agreement">W3C Member Agreement</a> and 
+        <a href="https://www.w3.org/2021/Process-20211102">W3C Process</a> while taking into account all elements of the
+        W3C Process, is to specify solutions that support advertising while acting in the interests of users, in 
+        particular providing Privacy [1]. The Working Group welcomes participation from all W3C Members that may wish to
+        contribute, in particular, from browser users and vendors, OS vendors, application vendors, advertisers,
+        publishers, ad buyers, advertising platforms and intermediaries, privacy advocates, web application developers,
+        and other interested parties.
       </p>                                                                
 
+      <h5 id="mission-notes">[1] Privacy Note</h5>
```

Comment:
Is this section intended to be included @jwrosewell ?

Reply:
Not in the final version.